easybuild/easyconfigs/p/Paraview/Paraview-4.2.0-intel-2014b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'Paraview'
+version = '4.2.0'
+
+homepage = "http://www.paraview.org"
+description = "Paraview is a scientific parallel visualizer"
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+toolchainopts = {'usempi': False, 'pic': True}
+
+download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['http://www.paraview.org/paraview-downloads/%s' % download_suffix]
+sources = ["ParaView-v%(version)s-source.tar.gz"]
+
+patches = ['missingheader-%(version)s.patch']
+
+dependencies = [
+    ('Qt','4.8.6'),
+    ('Mesa', '10.4.5', '-Python-2.7.8'),
```

Comment:
@boegel: yeah, I already started to make this with 2015a, but I have never finished. Either the PR is too big, or it has lots of deps. I would like to finish this as it is, or abandon this PR and finish 2015a vesrion (which should also be bumped to 2015b, and OpenFOAM bumped to 2.4.X), and honestly this OpenFOAM feels for me a never ending story.